tests/functions/test_convert_stock.py

```diff
@@ -0,0 +1,7 @@
+from janitor import get_symbol
+
+
+def test_convert_stock():
```

Comment:
Could you add a docstring here too, please? The intent behind the test should be documented. 

I can foresee one failure mode for this test - if the `GME` symbol gets assigned to a different name, then the test will fail. Can you add to the docstring to which website we should go to see what the expected return string ought to be? That would go a long way to ensuring long-term maintainability of the function.

Reply:
**Action item:** As per comment above.